docs/samples/graph/bgtest/bgtest/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
+)
+
+type Instance struct {
+	Name   string `json:"name"`
+	IntVal int    `json:"intval"`
+	StrVal string `json:"strval"`
+}
+
+type Response struct {
+	Source    string     `json:"from"`
+	Instances []Instance `json:"instances"`
+}
+
+func main() {
+	defaultURL := fmt.Sprintf("/v1/models/%s:predict", os.Getenv("target"))
+	r := gin.Default()
+	r.POST("/splitter", handle)
+	r.POST("/switch", handle)
+	r.POST("/single", handle)
+	r.POST("/ensemble", handle)
```

Comment:
seems like they are all using the same handle, should we just use the default url which is following the protocol ?